code-challenges.test.js

```diff
@@ -13,37 +13,132 @@
-const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
 // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
 
+        // FAIL./ code - challenges.test.js
+        // arrayShuffler
+        //     ✕ takes in an array, removes the first item from the array and shuffles the remaining content.
+
+        //   ● arrayShuffler › takes in an array, removes the first item from the array and shuffles the remaining content.
+
+        //     ReferenceError: arrayShuffler is not defined
+
+
 
 // b) Create the function that makes the test pass.
 
+// --> Pseudo Code <--
+// Create function called arrayShuffler
+// Takes in array
+// First .shift to remove 1st item in array
+// Iterate the rest of array and randomize with math.floor & random
+// Return the array at the end
+
+const arrayShuffler = (array) => {
+    array.shift()
+    for (i = 0; i < array.length; i++) {
+        let randomize = Math.floor(Math.random() * array.length)
+        let newArray = array[i]
+        array[i] = array[randomize]
+        array[randomize] = newArray
+    }
+    return array
+}
+
+        // PASS./ code - challenges.test.js
+        // arrayShuffler
+        //     ✓ takes in an array, removes the first item from the array and shuffles the remaining content. (1 ms)
 
 // --------------------2) Create a function that takes an array of numbers and returns an array of the minimum and maximum numbers in that order.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const nums1 = [3, 56, 90, -8, 0, 23, 6]
+describe('minMax', () => {
+    const nums1 = [3, 56, 90, -8, 0, 23, 6]
+    const nums2 = [109, 5, 9, 67, 8, 24]
+    it('takes an array of numbers and returns an array of the minimum and maximum numbers in that order.', () => {
+        expect(minMax(nums1)).toEqual(expect.arrayContaining([-8, 90]));
+        expect(minMax(nums2)).toEqual(expect.arrayContaining([5, 109]));
+    })
+})
+
 // Expected output: [-8, 90]
-const nums2 = [109, 5, 9, 67, 8, 24]
 // Expected output: [5, 109]
 
+        // FAIL./ code - challenges.test.js
+        // minMax
+        //     ✕ takes an array of numbers and returns an array of the minimum and maximum numbers in that order. (1 ms)
+
+        //   ● minMax › takes an array of numbers and returns an array of the minimum and maximum numbers in that order.
+
+        //     ReferenceError: minMax is not defined
 
 // b) Create the function that makes the test pass.
 
+// --> Pseudo Code <--
+// Create function called minMax
+// Set variable with the min value in array with math.min
+// Set variable with max value with math.max
+// return a result that is a combo of values
+
+const minMax = (array) => {
+    let minimum = Math.min(...array)
+    let maximum = Math.max(...array)
+    let result = ([minimum, maximum])
+    return result
+}
```

Comment:
Great work there is a refactor here that is very clean-- recommend trying to figure it out.